css/projects.css

```diff
@@ -0,0 +1,264 @@
+/* General CSS */
+
+body  {
+background-color: #E6DACD; 
+text-align:left;
+margin:0px;
+overflow-x: hidden;
+}
+
+a:link {
+    text-decoration: none;
+}
+
+h1{
+    font-weight:700;
+    font-size:2.5em;
+    padding-top: 3em;
+}
+
+h3 {
+    font-family: 'Poppins', sans-serif;
+    font-weight: 800;
+    font-size: 1.5em;
+    padding-left:5px;
+}
+
+/* CSS for Top Navigation */
+
+.hidex{
+    display:none;
+}
+
+.hamburger{
+    width:35px;
+    height:35px;
+    position:absolute;
+    top: 6px;
+    right: 6px;
+}
+
+
+.wrapsquare{
+    display:flex;
+    margin: 10px 0px 0px 0px;
+}
+
+.square {
+  height: 13.5px;
+  width: 13.5px;
+  background-color:#0150FD;
+  margin:9px 0px;
+}
+
+.developer {
+    font-weight:400;
+    font-size: 0.95em;
+    line-height: 2.2;
+}
+
+.toptext{
+    text-align:left;
+}
+
+
+
+header a{
+    font-size:.7em;
+}
+
+
+/* CSS for Profile Section */
+.jobposition{
+    font-weight:700;
+}
+.date{
+    color:#0150FD;
+    padding-top:3em;
+    font-weight:700;
+}
+
+.longtext{
+    
+    padding-top:2em;
+}
+.longtext2{
+    
+    padding-bottom:2.5em;
+}
+
+.bluesquare{
+    width:20px;
+    height:20px;
+}
+.bluesquare2{
+    width:10px;
+    height:10px;
+    margin:5px;
+}
+
+ul.bluelist{
+      list-style: none;
+
+}
+
+.wrapsquare{
+
+}
+
+.float {
+    background-color: white;
+    filter: drop-shadow(-10px 10px 3px lightgrey);
+    max-width:80%;
+    margin: 0 auto;
+    margin-bottom: 40px;
+}
+
+.roundimage {
+  border-radius: 50%;
+  padding: 20%;
+}
+
+.line{
+  height:2px;
+  width: 20%;
+  background-color:#0150FD ;  
+  margin: 0 auto;
+}
+
+.buttonrow{
+    padding:20px;
+}
+
+.button1{
+    background-color: #0150FD;
+    color:white;
+    border-radius: 20px;
+    padding: 3px 20px;
+    border:solid #0150FD 2px;
+    font-weight:600;
+    font-size:1.3em;
+    margin:10px 0px;
+}
+
+.button2{
+    border-radius: 20px;
+    padding: 3px 16px;
+    border:solid black 2px;
+    font-weight:600;
+    font-size:1.3em;
+    margin:10px 0px;
+}
+.capture{
+    width:100%;
+    height:100%;
+}
```

Comment:
Don't set a width AND a height on your `img`s as this will distort them. `width: 100%` should be enough, and let the height set itself naturally.

To better control the width of the image:
- wrap it inside a `div`
- give that `div` a width (and an optional height)
- and then just `width: 100%` on the `img` (no height)

Of course this can only work if your images are already the right aspect ratio. In this case, they probably aren't, so you could consider setting them as background images instead, which is easier to make fill the space using `background-size: cover`

Up to you! 🙂